src/asciidocUtils.ts

```diff
@@ -17,14 +15,15 @@ import { DidactWebviewPanel } from "./didactWebView";
  * limitations under the License.
  */
 
+// tslint:disable-next-line: typedef
 const asciidoctor = require('asciidoctor')();
 
-export function getADParser() {
+export function getADParser() : void {
```

Comment:
`void` is definitely wrong I'd say

Reply:
Yeah, you're probably right. :)

Reply:
turns out that function is never used, so we can remove it entirely